src/Tizen.NUI.Scene3D/src/internal/Interop/Interop.Model.cs

```diff
@@ -96,10 +96,10 @@ internal static partial class Model
             public static extern global::System.IntPtr LoadBvhAnimationFromBuffer(global::System.Runtime.InteropServices.HandleRef model, string bvhBuffer, int bvhBufferLength, global::System.Runtime.InteropServices.HandleRef scale, bool translateRootFromModelNode);
 
             [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_Model_LoadFacialAnimation_1")]
-            public static extern global::System.IntPtr LoadFacialAnimation(global::System.Runtime.InteropServices.HandleRef model, string facialFilename);
+            public static extern global::System.IntPtr LoadSamsungJsonMorphingAnimation(global::System.Runtime.InteropServices.HandleRef model, string jsonFilename);
```

Comment:
Actually, this is a kind of amazing technicolor dream naming.
We need to talk about this more....

Reply:
why don't you use LoadBlendShapeAnimation()?

Reply:
@bshsqa @hyunjushin  Thanks to both of you. I make the API name as `LoadBlendShapeAnimation`, and write some comment that we use json format file.